aikau/src/main/resources/alfresco/menus/_AlfMenuItemMixin.js

```diff
@@ -164,7 +164,7 @@ define(["dojo/_base/declare",
          }
          if (!this.iconAltText && this.title)
          {
-            this.iconAltText = originalLabel;
+            this.iconAltText = this.message(originalLabel);
```

Comment:
I think it is a pure semantic argument wether label or title should be the go-to fallback value for altText if not configured. I'd expect the label to be used for altText of the icon whereas title would be expected to be used in the DOM title attributes of both the label and the icon for the menu item.